roles/wordpress-install/tasks/main.yml

```diff
@@ -37,6 +37,27 @@
   when: item.value.site_install == True and (item.value.multisite.enabled | default(False) == False)
   changed_when: "'WordPress is already installed.' not in wp_install_results.stdout"
 
+- name: Check Existing Permalink Structure
+  command: wp rewrite list --format=count
+           --allow-root
+           --url="{{ item.value.env.wp_home }}"
+  args:
+    chdir: "{{ www_root }}/{{ item.key }}/current/"
+  register: wp_permalink_check_results
+  with_dict: wordpress_sites
+  when: item.value.site_install and not item.value.multisite.enabled | default(false)
+  changed_when: False
+
+- name: Setup Permalink Structure
+  command: wp rewrite structure {{ item.value.permalink_structure | default("/%postname%/") }}
+           --allow-root
+           --url="{{ item.value.env.wp_home }}"
+  args:
+    chdir: "{{ www_root }}/{{ item.key }}/current/"
+  register: wp_permalink_results
+  with_dict: wordpress_sites
+  when: "'No rewrite rules' in wp_permalink_check_results.results.0.stderr"
```

Comment:
@ckovey if the "Check Existing Permalink Structure" task skips, the variable `wp_permalink_check_results.results.0.stderr` is undefined, resulting in the following error for the "Setup Permalink Structure" task ([example](https://discourse.roots.io/t/trellis-multisite-install-fails-on-conditional-no-rewrite-rules/5627)):
```
TASK: [wordpress-install | Setup Permalink Structure] *************************
fatal: [default] => error while evaluating conditional: 'No rewrite rules' in wp_permalink_check_results.results.0.stderr
```

This error could be avoided with a more elaborate `when:` condition for the "Setup Permalink Structure" task:
```
when: item.value.site_install and not item.value.multisite.enabled | default(false) and 'No rewrite rules' in wp_permalink_check_results.results.0.stderr
```
The above prepends the conditional logic from the "Check Existing Permalink Structure", which is probably preferable by being more intuitive than the shorter option below:
```
when: wp_permalink_check_results.results.0.stderr is defined and 'No rewrite rules' in wp_permalink_check_results.results.0.stderr
```